vortex-expr/README.md

```diff
@@ -0,0 +1,7 @@
+# Vortex Expressions
+
+Expressions for querying vortex arrays, designed to express a minimal
```

Comment:
A crate defining serializable predicate expressions. Used predominantly for filter push-down.